source/extensions/filters/network/rocketmq_proxy/config.cc

```diff
@@ -0,0 +1,68 @@
+#include "extensions/filters/network/rocketmq_proxy/config.h"
+
+#include <cstdlib>
+
+#include "envoy/config/filter/network/rocketmq_proxy/v2alpha1/rocketmq_proxy.pb.h"
+#include "envoy/registry/registry.h"
+#include "envoy/server/filter_config.h"
+
+#include "extensions/filters/network/rocketmq_proxy/conn_manager.h"
+#include "extensions/filters/network/rocketmq_proxy/stats.h"
+#include "extensions/filters/network/well_known_names.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace RocketmqProxy {
+
+namespace rocketmq_config = envoy::config::filter::network::rocketmq_proxy::v2alpha1;
+
+Network::FilterFactoryCb RocketmqProxyFilterConfigFactory::createFilterFactoryFromProtoTyped(
+    const rocketmq_config::RocketmqProxy& proto_config,
+    Server::Configuration::FactoryContext& context) {
+  std::shared_ptr<ConfigImpl> filter_config = std::make_shared<ConfigImpl>(proto_config, context);
+  return [filter_config, &context](Network::FilterManager& filter_manager) -> void {
+    filter_manager.addReadFilter(
+        std::make_shared<ConnectionManager>(*filter_config, context.dispatcher().timeSource()));
+  };
+}
+
+REGISTER_FACTORY(RocketmqProxyFilterConfigFactory,
+                 Server::Configuration::NamedNetworkFilterConfigFactory);
+
+const char* ConfigImpl::ENV_APP_VERSION = "ENVOY_APP_VERSION";
+
+ConfigImpl::ConfigImpl(const RocketmqProxyConfig& config,
+                       Server::Configuration::FactoryContext& context)
+    : context_(context), stats_prefix_(fmt::format("rocketmq.{}.", config.stat_prefix())),
+      stats_(RocketmqFilterStats::generateStats(stats_prefix_, context_.scope())),
+      develop_mode_(config.develop_mode()) {
+  char* appVersionPtr = std::getenv(ENV_APP_VERSION);
```

Comment:
Add more comments

Reply:
This part would be removed with AppLoadBalancer.